File: Leetcode-Pattern-problems-main/Leetcode-Pattern-problems-main/input_format.cpp
#include <iostream>
#include <vector>

int main() {
    int t; // Number of test cases
    std::cin >> t;

    for(int i = 0; i < t; ++i) {
        int n; // Size of the array
        std::cin >> n;

        std::vector<int> arr(n);
        for(int j = 0; j < n; ++j) {
            std::cin >> arr[j];
        }

        // Process the array as needed
        for(int num : arr) {
            std::cout << num << " ";
        }
        std::cout << std::endl;
    }

    return 0;
}

File: Leetcode-Pattern-problems-main/Leetcode-Pattern-problems-main/input_format.py
t = int(input("Enter number of test cases: "))  # Number of test cases

for _ in range(t):
    n = int(input())  # Size of the array
    arr = list(map(int, input().split()))  # Read the array

    # Process the array as needed
    for num in arr:
        print(num, end=" ")
    print()